src/tests/test_aladin.py

```diff
@@ -239,3 +239,47 @@ def test_add_table(monkeypatch: Callable) -> None:
         "conversion_maj_axis": 1,
     }
     assert table_sent_message["options"]["ellipse_error"] == ellipse_options
+
+
+test_overlay_names = [
+    "overlay",
+    ["overlay", "overlay_1", "2MASS"],
+    "catalog",
+    ["overlay_1", "2MASS"],
+]
+
+
+@pytest.mark.parametrize("overlay_names", test_overlay_names)
+def test_remove_overlay(
+    monkeypatch: Callable,
+    overlay_names: Union[Iterable[str], str],
+) -> None:
+    """Test proper messages sent for removing overlays using their name string(s).
+
+    Parameters
+    ----------
+    overlay_names : Union[Iterable[str], str]
+        The name strings of overlays.
+
+    """
+    mock_send = Mock()
+    monkeypatch.setattr(Aladin, "send", mock_send)
+    aladin.remove_overlay(overlay_names)
+    name_info = mock_send.call_args[0][0]["name"]
+    assert isinstance(name_info, list)
+    assert name_info[0] in overlay_names
+
```

Comment:
we should also assert that the right method is called `"event_name": "remove_overlay"`